examples/tensorflow/classification/main.py

```diff
@@ -279,9 +282,12 @@ def export(config):
                                     pretrained=config.get('pretrained', False),
                                     weights=config.get('weights', None))
     model = model(**model_params)
-    compression_ctrl, compress_model = create_compressed_model(model,
-                                                               config.nncf_config,
-                                                               should_init=False)
+    compression_state = TFCompressionState()
+    if config.ckpt_path is not None:
+        checkpoint = tf.train.Checkpoint(compression_state=compression_state)
+        load_checkpoint(checkpoint=checkpoint, ckpt_path=config.ckpt_path)
```

Comment:
Discussed and decided leave as is, since a single line can be saved only with hiding API behind one more helper.

Instead of
```
compression_state = TFCompressionState()
if resume_training:
    checkpoint = tf.train.Checkpoint(compression_state=compression_state)
    load_checkpoint(checkpoint, config.ckpt_path)
```

there will be 
```
compression_state = TFCompressionState()
if resume_training:
    load_compression_state_from_checkpoint(compression_state, config.ckpt_path)
```
or 
```
compression_state = None
if resume_training:
    compression_state = load_compression_state_from_checkpoint(config.ckpt_path)
```